src/notation/view/percussionpanel/percussionpanelpadmodel.cpp

```diff
@@ -74,7 +81,45 @@ const QVariant PercussionPanelPadModel::notationPreviewItemVariant() const
     return QVariant::fromValue(m_notationPreviewItem);
 }
 
+QList<QVariantMap> PercussionPanelPadModel::footerContextMenuItems() const
+{
+    const muse::TranslatableString duplicatePadTitle("global", "Duplicate");
```

Comment:
Unfortunately, this way, the strings won't be detected by `lupdate`. The least painful alternative that does work is 
```suggestion
    const auto duplicatePadTitle = muse::TranslatableString("global", "Duplicate");
```
(and I think you can be reasonably certain that the compiler will optimise it so that it becomes basically the same).

Also, you could consider making these `static const` and making the `const int`s `static constexpr`. That might be slightly beneficial.

Reply:
Ah interesting - T.I.L!

Fixed now